app/src/main/java/com/example/splendormobilegame/config/Config.java

```diff
@@ -0,0 +1,48 @@
+package com.example.splendormobilegame.config;
+
+import android.content.Context;
+
+import com.example.splendormobilegame.config.exceptions.InvalidConfigException;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+
+public class Config implements IConfig {
+
+    private String serverip;
+    private Context context;
+    public Config(Context context)
+    {
+        this.context = context;
+
+    }
+
+
+    private void loadFile() throws InvalidConfigException {
+        Properties properties = new Properties();
+        try {
+            InputStream inputStream = context.getAssets().open("conig.properties");
```

Comment:
I've run app at this commit. And it didn't work. I suppose this line causes errors.